app/models/api/tta.rb

```diff
@@ -44,7 +44,7 @@ def path
     "/api/v1/activity-reports/display/#{display_id}"
   end
 
-  def query
-    ""
+  def parameters
+    []
```

Comment:
[issue(?)] should this be an empty hash? The comment in `ApiRequest` indicates that it should be a hash

Reply:
Oh snap, good catch yes, it's a hash -- I should change that and the tests stubbing that in (even though it's not used in them)